dashboard/prometheus/README.md

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
this file shouldn't be in this PR

Reply:
you also have two duplicate commit messages for this file 8d8ba2e and 5c517e1

Reply:
agree, removed